src/app/series/directives/series-feed.component.spec.ts

```diff
@@ -23,7 +23,7 @@ describe('SeriesFeedComponent', () => {
     comp.isLoaded = true;
     fix.detectChanges();
 
-    expect(el).toContainText('You have no published episodes');
+    expect(el).toContainText('You have no episodes');
```

Comment:
This text wasn't quite accurate, since this page shows published + released + draft episodes.